views/produto/admin_editar.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/* @var $produto Models\Produto */
+$produto = $data['produto'];
+?>
+
+<h3>Editar Produto</h3>
+<form method="POST" action="">
+    <input type="hidden" name="idProduto" value="<?= $produto->getIdProduto() ?>"/>
+    <div class="form-group">
+        <label for="titulo">Nome</label>
+        <input type="text" name="nome" id="nome" class="form-control" value="<?= $produto->getNome() ?>" placeholder="Nome"/>
+    </div>
+    <div class="form-group">
+        <label for="titulo">Descricao</label>
```

Comment:
o atributo `for` das labels estão sempre apontando pra "titulo" e não para os campos certos